R/get_metadata.R

```diff
@@ -1,26 +1,15 @@
 #' Get Water System
-#' @description Enable user to discover water system ID when supplied a zipcode or county of interest
-#' @param zipcode zipcode
+#' @description Enable user to discover water system ID when supplied a county of interest
 #' @param county county
 #' @examples
-#' get_water_system(zipcode = "90201")
 #' get_water_system(county = "D")
```

Comment:
we will need to update this documentation to use a real county after we merge Rich's data objects